docs/get-started/setup.md

```diff
@@ -4,25 +4,16 @@ title: 'Setup'
 
 Now that you’ve learned what stories are and how to browse them, let’s demo working on one of your components. Pick a simple component from your project, like a Button, and write a `.stories.js` file to go along with it. It might look something like this:
 
-```js
-// YourComponent.stories.js
+<!-- prettier-ignore-start -->
 
-import { YourComponent } from './YourComponent';
+<CodeSnippets
+  paths={[
+    'common/your-component.js.mdx',
+    'common/your-component.ts.mdx',
```

Comment:
Isn't this snippet react-specific? (has rendering of the template)

Reply:
the ts file is actually extracted from the angular, i was aiming for a "dummy example" for both js and ts options. Didn't want to go into specifics. If need be i'll adjust the examples to become react /react with ts and for angular. 

Let me know.